src/core/io/src/4C_io_vtk_writer_base.hpp

```diff
@@ -36,13 +36,26 @@ namespace LibB64
    */
   char* encode_block(const char* data, const int data_size);
 
+  // This enum class encodes the compression levels available in zlib. Usually, you want to use
+  // best_speed for compression since it offers nearly as good compression as best_compression while
+  // being a lot faster (up to six times). best_speed is the default value if it is not declared
+  // otherwise in the input file.
```

Comment:
```suggestion
 /**
  * This enum class encodes the compression levels available in zlib. Usually, you want to use
  * best_speed for compression since it offers nearly as good compression as best_compression while
  * being a lot faster (up to six times).
  */
```